source/auth/auth.md

```diff
@@ -1682,14 +1685,15 @@ authenticate a new connection when a [OIDC Human Callback](#oidc-human-callback)
 
 - Check if the *Client Cache* has an access token.
   - If it does, cache the access token in the *Connection Cache* and perform a [One-Step](#one-step) SASL conversation
-    using the access token. If the server returns an error, invalidate the access token token from the *Client Cache*,
-    clear the *Connection Cache*, and continue.
+    using the access token. If the server returns an Authentication error (18), invalidate the access token token from
+    the *Client Cache*, clear the *Connection Cache*, and continue. If the server returns another error, continue.
```

Comment:
See prior comment regarding discussion - this again might not be the right PR, but it seems we should clarify the use of "continues", and should also change the logic to use explicit looping (to avoid the example of 20 redundant invocations of a callback when a token expires across 21 connections).

Reply:
Updated